tests/Microsoft.Identity.Test.Unit/WebUITests/NetCoreWebUIFactoryTests.cs

```diff
@@ -54,6 +54,9 @@ public void NetCoreFactory_Embedded()
         }
 
         [TestMethod]
+#if ONEBRANCH_BUILD
```

Comment:
it is failing because of this method. https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/546d00f3cbf52e8ca035a789736d9d802be0e5d7/src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/DesktopOsHelper.cs#L157
Not exactly sure what it is it.